.travis.yml

```diff
@@ -1,31 +1,54 @@
+sudo: required
```

Comment:
Please be more careful with your commits/branch creation. These changes to .travis.yml aren't changes you've made, but ones you've pulled from somewhere else. Pull .travis.yml from master.